graphql/inputs/wallet-inputs.graphql

```diff
@@ -0,0 +1,11 @@
+""" Input for creating a wallet """
+input CreateWalletInput {
```

Comment:
Delete the inputs for creating/updating a wallet. Wallets MUST only be changed when transactions are carried out on them

Reply:
Done